packages/tc-schema-validator/tests/enums.js

```diff
@@ -1,7 +1,8 @@
 /* global it, describe, expect */
 const { SDK } = require('@financial-times/tc-schema-sdk');
+const readYaml = require('@financial-times/tc-schema-sdk/lib/read-yaml');
```

Comment:
and remove this import.

The reason this is a good thing to do is because unless a library explicitly documents additional files are meant for requiring, it's safer/less likely to need refactoring to just require the package by name, rather than files internal to the package.

Reply:
thanks